2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,72 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let recipe1 = {
+  title: "Broccoli and bacon salad",
+  serving: 2,
+  ingredients: {
+    bacon: "150g",
+    broccoli: "800g",
+    mayonnaise: "125g (1/2 cup)",
+    "fresh lemon juice": "1 tbsp",
+    mustard: "2 tsp",
+    "red grapes": "200g",
+    "green shallots": 2,
+    "slivered almonds": "2 tbsp",
+  },
+};
+
+let recipe2 = {
+  title: "Prawn cocktail",
+  serving: 4,
+  ingredients: {
+    prawn: "400 g",
+    "Little Gem lettuces": 4,
+    mayonnaise: "5 heaped tbsp",
+    "tomato chutney": "5 tbsp ",
+  },
+};
+let recipe3 = {
+  title: "Pastrami-cured salmon",
+  serving: 6,
+  ingredients: {
+    salmon: "800 g",
+    "sea salt": "250 g",
+    sugar: "100 g",
+    coriander: "2 tbsp",
+  },
+};
+
+let recipe4 = {
+  title: "Roast turkey",
+  serving: 8,
+  ingredients: {
+    turkey: "5-6 kg",
+    butter: "150 g",
+    "garlic cloves": "3",
+    "mixed herbs such as parsley, thyme or bay": "optional",
+    lemons: 2,
+    onion: 1,
+    carrots: 2,
+  },
+};
+
+let recipe5 = {
+  title: "Really simple cranberry sauce",
+  serving: 8,
+  ingredients: {
+    sugar: "100 g",
+    "orange juice": "100 ml",
+    cranberries: "250 g",
+  },
+};
+
+myRecipes = [recipe1, recipe2, recipe3, recipe4, recipe5];
+myRecipes.forEach((x) => console.log(x));
+// console.log(recipe1);
```

Comment:
great job, here we wanted you to show each "key" inside of each recipe, instead of seeing the whole recipe, can you think of how to do this? 
I can add my solution here for you but please try your own solution for this : 
`myRecipes .map(eachRecipe => {console.log(eachRecipe.title);console.log(eachRecipe.Serves);console.log(eachRecipe.Ingredients.forEach(eachIngredients => console.log(eachIngredients)));})`

Reply:
thank you for your comment! I really appreciate this, my understanding of the task was a little bit different. I am trying to solve now in a correct way. I've tried your solution and it says "   eachRecipe.Ingredients.forEach((eachIngredients) =>
                           ^

TypeError: Cannot read property 'forEach' of undefined". Do you have any suggestions why is so? I cannot understand why it doesn't work, as it seems to be written correctly.